src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -126,52 +127,58 @@ def handle_message(self, message):
         :param message:
         :return:
         """
-        try:
-            contents = self.parse_message_value(message.value())
-        except InvalidMessageError:
-            # If the message is in an invalid format, just log the error
-            # and continue
-            logger.exception(
-                "Subscription update could not be parsed",
-                extra={
-                    "offset": message.offset(),
-                    "partition": message.partition(),
-                    "value": message.value(),
-                },
-            )
-            return
-
-        try:
-            subscription = QuerySubscription.objects.get_from_cache(
-                subscription_id=contents["subscription_id"]
-            )
-        except QuerySubscription.DoesNotExist:
-            metrics.incr("snuba_query_subscriber.subscription_doesnt_exist")
-            logger.error(
-                "Received subscription update, but subscription does not exist",
-                extra={
-                    "offset": message.offset(),
-                    "partition": message.partition(),
-                    "value": message.value(),
-                },
-            )
-            return
-
-        if subscription.type not in subscriber_registry:
-            metrics.incr("snuba_query_subscriber.subscription_type_not_registered")
-            logger.error(
-                "Received subscription update, but no subscription handler registered",
-                extra={
-                    "offset": message.offset(),
-                    "partition": message.partition(),
-                    "value": message.value(),
-                },
-            )
-            return
-
-        callback = subscriber_registry[subscription.type]
-        with metrics.timer("snuba_query_subscriber.callback.duration", instance=subscription.type):
-            callback(contents, subscription)
+        with sentry_sdk.Hub.current.push_scope() as scope:
```

Comment:
can be `sentry_sdk.push_scope()`

Reply:
Nice, wasn't sure if I needed the Hub or not